### DISCUSSION TITLE
[1차 배포] 배포 관련 모든 정보를 모아봅시다!

### DISCUSSION THREAD

**Discussion by jinmoon23** [SwiftBreakers/TTEOPPOKKI4U#52] [Ideas]
https://appstoreconnect.apple.com/ <- 배포를 위한 스크린샷 업로드 및 설명을 입력합니다. 또한 TestFlight 관련 설정도 여기서 진행합니다. 
https://sodevly.github.io/react-native-upload-app-on-testflight/ <- TestFlight 배포 관련 정보입니다. 
https://github.com/socar-abel/ZZik-Call/blob/main/개인정보처리방침.md <- 개인정보처리방침에 대한 예시입니다.

**Comment by labydin** [comment]
https://velog.io/@heyksw/iOS-%EC%95%B1%EC%8A%A4%ED%86%A0%EC%96%B4-%EB%B0%B0%ED%8F%AC%EA%B3%BC%EC%A0%95  <- 앱스토어 배포 과정
https://velog.io/@s_sub/%EC%83%88%EC%8B%B9-iOS-14%EC%A3%BC%EC%B0%A8 <- TestFlight 및 앱스토어 심사 제출 과정(이분 글에 실시간 채팅 로직 구현 후기도 있어요!)
https://documentation.swing2app.co.kr/knowledgebase/appstore/infor  <- 앱스토어 앱 등록 시 준비사항
http://redballs.tistory.com/entry/%EC%95%84%EC%9D%B4%ED%8F%B0-%EC%95%B1-%EA%B2%80%EC%88%98%EB%A5%BC-%EC%9C%84%ED%95%9C-%EC%95%A0%ED%94%8C-%EB%A7%B5-Apple-Map-%EC%97%B0%EB%8F%99  <-  길찾기 기능 네이버맵만 연결했다가 리젝먹은 사례
https://youngkdevlog.tistory.com/47  <- 어떤 화면인지 알 수 없으니 설명해달라는 리젝 사유(더미데이터가 없어서 심사관이 심사를 할 수 없었음)
